debug/tinysdf.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Mapbox GL JS debug page</title>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
+    <link rel='stylesheet' href='/dist/mapbox-gl.css' />
+    <style>
+        body { margin: 0; padding: 0; }
+        html, body, #originalMap, #newMap { height: 100%; }
+        #originalMap, #newMap { width: 49%; }
+
+        #originalLabel {
+            position: absolute;
+            background: #fff;
+            top:0;
+            left:0;
+            padding:10px;
+        }
+
+        #newLabel {
+            position: absolute;
+            background: #fff;
+            top:0;
+            right:0;
+            padding:10px;
+        }
+
+        #checkboxes {
+            position: absolute;
+            background: #fff;
+            bottom:0;
+            left:0;
+            padding:10px;
+        }
+
+    </style>
+</head>
+
+<body>
+
+<div id='originalMap' style="float: left"></div>
+<div id='newMap' style="float: right"></div>
+<label id='originalLabel'>Original Map</label>
+<label id='newLabel'>New Map</label>
+<div id='checkboxes'>
+    <label><input id='show-overdraw-checkbox' type='checkbox'> overdraw debug</label><br />
+</div>
+
+<script src='/dist/mapbox-gl-dev.js'></script>
+<script src='/debug/access_token_generated.js'></script>
+<script>
+
+var originalMap = window.originalMap = new mapboxgl.Map({
+    container: 'originalMap',
+    zoom: 8.8,
+    center: [121.574, 31.1489],
+    style: '/debug/streets-zh-global-v1.json',
+    hash: true
+});
+
+var newMap = window.newMap = new mapboxgl.Map({
+    container: 'newMap',
+    zoom: 8.8,
+    center: [121.574, 31.1489],
+    style: '/debug/streets-zh-global-v1.json',
+    localIdeographFontFamily: '"Noto Sans", "Noto Sans CJK SC", sans-serif',
```

Comment:
@1ec5, do you have an opinion on the naming here? Here was my reasoning for changing the original "CJK" to "Ideograph":

- "CJK" is an acronym that requires some contextual awareness (although in our context it's probably not too mysterious), while "ideograph" is a plain (?) English word.
- "CJK" implies that it's based on language, when it's really based on whether fixed-sized ideographs are being displayed (for instance, the Hiragana block for Japanese isn't included, as well as latin letters, etc.). Deciding to include the precomposed 'Hangul Syllables' block made this name a bit less appropriate, because they're not really ideographs, they just behave like ideographs.